core/sile.lua

```diff
@@ -119,6 +120,9 @@ Options:
   if opts.include then
     SILE.preamble = type(opts.include) == "table" and opts.include or { opts.include }
   end
+  if opts.traceback then
+    SILE.printTraceback = true
```

Comment:
What do you think about making this _the print traceback function_ instead of a boolean flag?

```lua
SILE.printTraceback = options.traceback and debug.traceback or print
```

This would possibly give us a place to hook into with a custom function instead.